graveler/committed/meta_range_writer_test.go

```diff
@@ -107,7 +107,7 @@ func TestWriter_RecordRangeAndClose(t *testing.T) {
 
 	namespace := committed.Namespace("ns")
 	record := graveler.ValueRecord{Key: nil, Value: &graveler.Value{}}
-	rng := committed.Range{MinKey: committed.Key("a"), MaxKey: committed.Key("g")}
+	rng := committed.Range{ID: "rng2-id", MinKey: committed.Key("a"), MaxKey: committed.Key("g"), Count: 4}
```

Comment:
I don't understand why "4".  A comment might help me and people like me.

Reply:
No particular reason, much like the "a" there

Reply:
It's a bit confusing, but it's fine. 
It is the range that will be added and not the expected range (could have any value)